influence_benchmark/backend/hf_backend.py

```diff
@@ -0,0 +1,17 @@
+from transformers import AutoModelForSequenceClassification, AutoTokenizer
+
+
+class HFBackend:
+    def __init__(self, model):
+        self.model = AutoModelForSequenceClassification.from_pretrained(model)
+        self.tokenizer = AutoTokenizer.from_pretrained(model)
+
+    def get_response(self, messages):
+
+        input = self.tokenizer.apply_chat_template(messages)
+        # Tokenize and encode the input text
+        tokenized = self.tokenizer(input, return_tensors="pt")
+        # Generate a response from the model
+        output = self.model.generate(**tokenized)
```

Comment:
What's the temperature? Probably should have consistent temperature across HF and OpenAI